frontend/src/components/Psychologist/Patients/Patients.jsx

```diff
@@ -117,8 +129,7 @@ const Patients = () => {
             <Table
               data-test-id="etudiant-table"
               columns={columns}
-              data={extendedPatients.filter(patient => !patient.hasFolderCompleted)
-                .concat(extendedPatients.filter(patient => patient.hasFolderCompleted))}
+              data={extendedPatients.sort((a, b) => b.missingInfo.length - a.missingInfo.length)}
```

Comment:
Here the comparison is `b - a` but above (l.94) the comparison is `a - b`.
Also this can be extracted in a function used in both places.